### ISSUE TITLE
Messages from bus-proxyd (kdbus) when polkit changes

### ISSUE THREAD

**Issue by heftig** [systemd/systemd#545]
Whenever a polkit auth agent registers or unregisters itself, I get one of these in my logs:

```
Send permission check kernel to dbus-1 for
  uid=102
  gid=102
  message=signal
  name=org.freedesktop.ModemManager1
  path=/org/freedesktop/PolicyKit1/Authority
  interface=org.freedesktop.PolicyKit1.Authority
  member=Changed
  : DENY
```

`uid=102` is polkit. I think this happens because ModemManager's policy starts with
`<deny send_destination="org.freedesktop.ModemManager1"/>`. This is probably needlessly restrictive, as the default bus policy already rejects all method calls.

**Comment by dvdhrm** [comment]
Yes, your analysis sounds correct to me. I now reviewed dbus-daemon, as I assume from your description that this warning only occurs on kdbus, not on dbus1, right?

So dbus-daemon prints the same warnings if a policy denies a transaction. However, dbus-daemon only prints this if the actual recipient matches the addressed recipient . This suppresses those message in case of eavesdropping or broadcasts (as in both cases the actual recipient is not mentioned in the message itself).

I'm not sure why dbus-daemon suppresses the messages in those cases, but I assume it's a compromise as otherwise it would spam the log due to widely opened MATCH rules of some applications.

Now, I wouldn't mind doing the same for bus-proxy. However, I'd prefer delaying that change, as those messages (even in this case) are highly useful to debug bus-proxy. And I actually _want_ people to report such false warnings. Furthermore, I think someone should seriously tell the ModemManager people that a rule of `<deny send_destination="org.freedesktop.ModemManager1"/>` prevents them from receiving **any** signal of another process (even if they subscribed to it).
(which I now did: https://bugs.freedesktop.org/show_bug.cgi?id=91347)

I'm closing this now. We can re-open this if ModemManager people reject the patch.
